corehq/apps/app_manager/templates/app_manager/v2/partials/build_errors.html

```diff
@@ -12,9 +12,9 @@ <h4 class="alert-heading">
                     <span>
                         {% case error.type "blank form" %}
                             {% if not error.message %}
-                                <i class="fa fa-plus"></i> <strong>Add Question</strong> to the
+                                <strong>Add a Question</strong> to the
```

Comment:
I wasn't sure if there was a reason the build errors were not translated @orangejenny